.travis.yml

```diff
@@ -12,9 +12,7 @@ jdk:
 matrix:
   allow_failures:
   - jdk: oraclejdk9
-  - jdk: oraclejdk10
   - jdk: openjdk9
```

Comment:
may as well either remove 9 entirely or remove it from allow_failures too

Reply:
Theory was that as the MSA is run by RPs they might have to run it on JDK9 if that's the version they support locally.  This would apply to the VSP too.  In Hub there would only ever be one version required at a time as it only runs here.

Reply:
in that case might as well remove it from allow_failures too